querydsl-apt/pom.xml

```diff
@@ -79,6 +79,15 @@
       <artifactId>morphia</artifactId>
       <version>${morphia.version}</version>
       <scope>provided</scope>
+    </dependency>
+    <dependency>

```

Comment:
I suggest we remove the logging from this class, I cannot see any other logging in apt.